avocado/core/job.py

```diff
@@ -564,7 +570,6 @@ def run(self):
             self.exitcode |= exit_codes.AVOCADO_FAIL
             return self.exitcode
         finally:
-            self.job_pre_post_dispatcher.map_methods('post', self)
```

Comment:
Yes, it had been undefined for a reason, we wanted to get some real-world usage before defining it in documentation. Your definition looks reasonable so we can first define it in the `job` and after a while, if everyone is happy, we should finally document and support it.